.travis.yml

```diff
@@ -56,16 +56,17 @@ matrix:
         - RAY_GCS_SERVICE_ENABLED=false
       install:
         - eval `python $TRAVIS_BUILD_DIR/ci/travis/determine_tests_to_run.py`
+        - if [ $RAY_CI_SERVE_AFFECTED != "1" ] && [ $RAY_CI_TUNE_AFFECTED != "1" ] && [ $RAY_CI_PYTHON_AFFECTED != "1" ] && [ $RAY_CI_STREAMING_CPP_AFFECTED != "1" ] && [ $RAY_CI_JAVA_AFFECTED != "1" ]; then exit; fi
```

Comment:
this condition doesn't work. It should run when `RAY_CI_JAVA_AFFECTED` is 1

Reply:
and it does, no?
It will NOT(!) exit the install, if RAY_CI_JAVA_AFFECTED=1

Then, in the script section: `if [ $RAY_CI_JAVA_AFFECTED == "1" ]; then ./java/test.sh; fi`